### ISSUE TITLE
Cannot read property 'scripts' of null

### ISSUE THREAD

**Issue by GabLeRoux** [abe33/atom-pigments#243]
I was getting the following error:
```
Failed to load the pigments package
Cannot read property 'scripts' of null in /home/gableroux/.atom/packages/pigments/package.json
Hide Stack Trace
TypeError: Cannot read property 'scripts' of null
    at normalize (/usr/share/atom/resources/app.asar/node_modules/normalize-package-data/lib/normalize.js:25:11)
    at PackageManager.module.exports.PackageManager.normalizePackageMetadata (/usr/share/atom/resources/app.asar/src/package-manager.js:727:16)
    at PackageManager.module.exports.PackageManager.loadPackageMetadata (/usr/share/atom/resources/app.asar/src/package-manager.js:701:18)
    at PackageManager.module.exports.PackageManager.loadPackage (/usr/share/atom/resources/app.asar/src/package-manager.js:423:36)
    at /usr/share/atom/resources/app.asar/src/package-manager.js:402:19
    at Config.module.exports.Config.transact (/usr/share/atom/resources/app.asar/src/config.js:312:16)
    at PackageManager.module.exports.PackageManager.loadPackages (/usr/share/atom/resources/app.asar/src/package-manager.js:397:19)
    at /usr/share/atom/resources/app.asar/src/atom-environment.js:759:28
  at /home/gableroux/.atom/packages/pigments/package.json:1:1
```

**Comment by GabLeRoux** [comment]
Seemed similar to #42 
So I opened a terminal console and executed the following:

```
apm install pigment
```

And now when I open Atom, I don't get this error anymore.

**Comment by abe33** [comment]
Hi @GabLeRoux, just for the record, do you remember in which context this error started to popup, was it during an update ?

**Comment by GabLeRoux** [comment]
No I installed it a few months ago, can't remember when exactly and I think it did this as soon as I installed it. I didn't mind at that time and kept the plugin anyway. 

I know I kept my plugins plugins up to date from the ui. I think it did this all the time until I decide to try to get rid of this error so not sure if this is something others will get, but I took the time to note what fixed this ;)

I usually open atom from the command line if it's somehow related. I know it did a difference for one of the golang package I use that uses environment variables.
